blockchain/message_pool/Cargo.toml

```diff
@@ -23,21 +23,22 @@ forest_message         = { version = "0.7", features = ["proofs", "blst"] }
 forest_networks        = "0.1.0"
 forest_state_manager   = "0.1.0"
 forest_vm              = "0.3.1"
-futures                = "0.3.5"
-fvm                    = "1.0"
+futures                = "0.3.24"
+fvm                    = "1.1"
 fvm_ipld_encoding      = "0.2"
 fvm_shared             = { version = "0.8.0", default-features = false }
-libsecp256k1           = "0.6"
-log                    = "0.4.8"
-lru                    = "0.7.2"
-num-rational           = "0.4"
-num-traits             = "0.2"
-rand                   = "0.8.3"
-serde                  = { version = "1.0", features = ["derive"] }
-slotmap                = "1.0"
-statrs                 = "0.15"
-thiserror              = "1.0"
-tokio                  = { version = "1.0", features = ["sync"] }
+libsecp256k1           = "0.7"
+log                    = "0.4.17"
+# cannot update lru as `LruCache::new` changed its arguments
```

Comment:
Can we adapt our usage then to this new API? It's only a number that got slightly more strict. If it's more complex for whatever reason then let's create a separate issue.

Reply:
A small issue would be better as there should be made some changes to the code

Reply:
Created